File: Application/Common/Dto/Item/ItemDto.cs
using Application.Common.Mappings;
using AutoMapper;
using Domain.Entities;
using System;
using System.Collections.Generic;

namespace Application.Common.Dto
{
    public class ItemDto : IMapFrom<Item>
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Information { get; set; }

        public IEnumerable<TagDto> Tags { get; set; }

        public IEnumerable<ImageDto> Images { get; set; }

        public DateTime CreationTime { get; set; }

        public CollectionNameDto Collection { get; set; }

        public UserLoginDto User { get; set; }

        void IMapFrom<Item>.Mapping(Profile profile) =>
            profile.CreateMap<Item, ItemDto>()
                   .ForMember(d => d.CreationTime, s => s.MapFrom(i => DateTime.SpecifyKind(i.CreationTime, DateTimeKind.Utc)));
    }
}

File: WebUI/ClientApp/src/app/models/dtos/item/item.dto.ts
import { CollectionNameDto } from "../collection/collection-name.dto";
import { ImageDto } from "../image.dto";
import { TagDto } from "../tag/tag.dto";
import { UserLoginDto } from "../user/user-login.dto";

export class ItemDto {
  public id: number;
  public name: string;
  public information: string;
  public tags: TagDto[];
  public images: ImageDto[];
  public creationTime: Date;
  public collection: CollectionNameDto;
  public user: UserLoginDto;
}